packages/x-date-pickers-pro/src/MultiInputDateTimeRangeField/MultiInputDateTimeRangeField.types.ts

```diff
@@ -51,10 +51,11 @@ export interface MultiInputDateTimeRangeFieldSlotsComponent {
    */
   Root?: React.ElementType;
   /**
-   * Input rendered for the start or end date.
+   * Component rendering an HTML input and a label.
+   * Will be rendered twice: once for the start date-time and once for the end date-time.
```

Comment:
I don't see us using `date-time` in written form, only for code or url parts. 🤔 
```suggestion
   * Will be rendered twice: once for the start date time and once for the end date time.
```